### ISSUE TITLE
Using the new "jump to leaderboard position" on any player from #1-99 create a duplicate leaderboard 

### ISSUE THREAD

**Issue by Kecxyz** [tetrio/issues#949]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this issue is not already reported.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am using a **supported browser** or the Desktop Client.
- [X] I am not using anything that **modifies game code** (e.g. TETR.IO PLUS).
- [X] I am certain this is a **bug**, not a feature request.

### OS and Version

iOS 15.1

### Browser and version

Safari for iOS 15.1

### Describe the bug/issue

When using the new "jump to leaderboard position" on any player in the rank rage of 1 to 99 will create a/multiple duplicate leaderboard.

### Reproducing

The fastest way to observe this, is to go to the current rank 1 player in Tetra League profile. Use the jump to leaderboard position by clicking on their rank. And see that there are now 2 of them on the leaderboard.

### Expected behavior

The leaderboard only show the player once

### Screenshots

![A743146B-84DB-4DF3-818A-A2573E096EA4](https://user-images.githubusercontent.com/41269576/169871024-01e09710-8b80-46ae-987d-40aaed3fb10f.png)

**Comment by o5k** [comment]
Cannot reproduce?

**Comment by Kecxyz** [comment]
![AFDAB165-F087-43EC-8B93-1E05A0DD02A3](https://user-images.githubusercontent.com/41269576/182662389-642f6be9-82ee-405a-8cc8-862c9cfa0e44.png)
Still reproducible on my end

**Comment by vojta-dev** [comment]
Here is a video of me reproducing the issue if that helps.

https://user-images.githubusercontent.com/65811166/182917256-11f93a70-5276-471a-9ed5-25c490bfe644.mp4

**Comment by vojta-dev** [comment]
this issue still happens 100% of the time, so still relevant

**Comment by lts20050703** [comment]
Can reproduce

<img width="960" alt="image" src="https://user-images.githubusercontent.com/87189679/235822536-20cfdec0-0d09-46d5-ae41-b73ae1e8ac41.png">

**Comment by Kecxyz** [comment]
It seems to be a problem with non standard aspect ratio / extremely small resolution 

https://github.com/tetrio/issues/assets/41269576/d6404481-97b9-4274-8a52-1c7d7afa1db3

**Comment by lts20050703** [comment]
My display is 1080p (1920x1080) BUT Windows recommend me to set display scaling to 150%
<img width="720" alt="image" src="https://github.com/tetrio/issues/assets/87189679/f2764c5e-32e9-48e5-a2ce-41542b9f3b4a">

**Comment by lts20050703** [comment]
Is there any update on this?

**Comment by lts20050703** [comment]
Still not fixed

**Comment by ZaptorZap** [comment]
Well, this one regressed. Considering the code changes were detailed in #998 instead, I'm going to close this one as a duplicate of that one.

**Comment by ZaptorZap** [comment]
No, wait, that one is definitely a unique issue. Either way this one doesn't happen anymore.

**Comment by Kecxyz** [comment]
![IMG_3857](https://github.com/tetrio/issues/assets/41269576/18f9573d-2757-4e29-9bb8-18fe4423872f)
Still reproducible on my end, now with messed up leaderboard standing

**Comment by ZaptorZap** [comment]
Well, doesn't happen to me........

**Comment by Kecxyz** [comment]
> It seems to be a problem with non standard aspect ratio / extremely small resolution
> 
>  boop.mp4

Definitely still reproducible using the window resolution from the video for me
![image](https://github.com/tetrio/issues/assets/41269576/91c38a27-1236-448e-b551-0851c8081858)

**Comment by lts20050703** [comment]
@zaptorzap Still reproducible![image](https://github.com/tetrio/issues/assets/87189679/b8f57722-e6b0-471d-8fde-b7ecdc8ca487)
